src/SecTester.Repeater/Dispatchers/HttpCommandDispatcher.cs

```diff
@@ -5,12 +5,12 @@
 using System.Net.Http.Headers;
 using System.Threading;
 using System.Threading.Tasks;
-using SecTester.Bus.Commands;
-using SecTester.Bus.Extensions;
 using SecTester.Core.Bus;
 using SecTester.Core.Utils;
+using SecTester.Repeater.Commands;
+using SecTester.Repeater.Extensions;
 
-namespace SecTester.Bus.Dispatchers;
+namespace SecTester.Repeater.Dispatchers;
 
 public class HttpCommandDispatcher : ICommandDispatcher
```

Comment:
This is not the appropriate place for such heavy abstractions and implementations. Additionally, it is unclear why the repeater is responsible for this. Move them to the core